pilot/pkg/bootstrap/configcontroller.go

```diff
@@ -175,7 +175,11 @@ func (s *Server) initK8SConfigStore(args *PilotArgs) error {
 					AddRunFunction(func(leaderStop <-chan struct{}) {
 						// We can only run this if the Gateway CRD is created
 						if configController.WaitForCRD(gvk.KubernetesGateway, leaderStop) {
-							tagWatcher := revisions.NewTagWatcher(s.kubeClient, args.Revision)
+							var tagWatcher revisions.TagWatcher
+							// TagWatcher requires permission for MutatingWebhook, so it can't be used in multi-tenant mode
+							if s.kubeClient.GetMemberRoll() == nil {
+								tagWatcher = revisions.NewTagWatcher(s.kubeClient, args.Revision)
+							}
```

Comment:
JFYI: This change is new comparing to previous releases.

Reply:
> I wonder how s.kubeClient.GetMemberRoll() being not nil means we are in the multi-tenant mode?

Being not nil means that `--memberRoll=default` argument was passed to istiod -> istiod watches ServiceMeshMemberRoll -> istiod runs in multi-tenant mode.